lib/hexpm_web/views/package_view.ex

```diff
@@ -156,4 +156,92 @@ defmodule HexpmWeb.PackageView do
         [content_tag(:strong, "Retired package")]
     end
   end
+
+  def pretty_date(datetime) do
+    "#{DateTime.to_date(datetime)}"
+  end
+
+  @doc """
+  This function turns an audit_log struct into a short description.
+
+  Please check Hexpm.Accounts.AuditLog.extract_params/2 to see all the
+  package related actions and their params structures.
+  """
+  def humanize_audit_log_info(%{action: "docs.publish"} = audit_log) do
```

Comment:
We might want to move this to the AuditLog schema module, because we will be showing audit logs in multiple places in the future.

We don't have to do this now, this is just a note for later.